### PR TITLE
ui:15636453 - Colony details section

### PR DESCRIPTION
**Pull request by krawczykkatarzyna**
- add base pager for dashboard 
- display title and description
- display placeholder when colony has no description
- display symbol with icon and tooltip
- display links, share icon and burger menu actions

NOTE:  opening notifications tab will be implemented when the card will be ready.

Screen Shot 2023-11-14 at 09 01 50

Screen Shot 2023-11-14 at 11 18 05

Screen Shot 2023-11-14 at 11 18 27

Screen Shot 2023-11-14 at 11 19 51

Screen Shot 2023-11-14 at 12 27 24

Screen Shot 2023-11-14 at 15 23 19

Screen Shot 2023-11-14 at 15 23 39

Screen Shot 2023-11-14 at 18 51 57

### CODE REVIEW

File: src/components/frame/v5/pages/DashboardPage/partials/Menu/Menu.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          withTooltipStyles={false}
+          cardProps={{
+            rounded: 's',
+            hasShadow: true,
+            className: 'py-4 px-2',
+          }}
+          classNames="w-full sm:max-w-[17.375rem]"
+        >
+          <Link
+            to={`/colony/${colony?.name}/details`}
```
Comment (CzarekDryl) [inline src/components/frame/v5/pages/DashboardPage/partials/Menu/Menu.tsx:55]: Can you use `COLONY_DETAILS_ROUTE` here?

File: src/components/frame/v5/pages/DashboardPage/partials/Menu/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            className="subnav-button"
+          >
+            <Icon name="custom-logo" appearance={{ size: 'extraSmall' }} />
+            <span className="ml-2">
+              {formatText({ id: 'dashboard.burderMenu.item.aboutColony' })}
```
Comment (CzarekDryl) [inline src/components/frame/v5/pages/DashboardPage/partials/Menu/Menu.tsx:60]: `burderMenu` => `burgerMenu`

File: src/components/v5/shared/BurgerMenu/BurgerMenu.tsx [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+}) => {
   const { formatMessage } = useIntl();
 
   return (
     <button
       type="button"
-      className="text-gray-400 hover:text-blue-400 p-[0.1875rem] transition-all duration-normal cursor-pointer"
+      className={clsx(
+        className,
+        'text-gray-400 hover:text-blue-400 p-[0.1875rem] transition-all duration-normal cursor-pointer',
```
Comment (CzarekDryl) [inline src/components/v5/shared/BurgerMenu/BurgerMenu.tsx:22]: sm:hover

File: src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        <ul className="flex items-center gap-4">
+          {externalLinks.map(({ name, link }) => (
+            <li key={name}>
+              <Link to={link} className={iconStyles}>
+                <Icon
+                  name={
+                    (name === 'Custom' && 'custom-logo') ||
+                    (name === 'Discord' && 'discord-logo') ||
+                    (name === 'Whitepaper' && 'whitepaper-logo') ||
+                    ''
```
Comment (CzarekDryl) [inline src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx:35]: Maybe if there is no name then don't render `<Icon />`?

File: src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+const Links: FC = () => {
+  const { colony } = useColonyContext();
+  const { metadata, colonyAddress } = colony || {};
+  const { externalLinks } = metadata || {};
+  const { handleClipboardCopy, isCopied } = useCopyToClipboard(
+    colonyAddress || '',
+    5000,
+  );
+
+  const iconStyles = 'flex items-center md:hover:text-blue-400';
```
Comment (CzarekDryl) [inline src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx:21]: It's more like `linkStyles` or `linkIconStyles`

File: src/components/frame/v5/pages/DashboardPage/partials/ColonyHeader/ColonyHeader.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+      : description;
+
+  return (
+    <header className="flex flex-col gap-4">
+      <div className="flex justify-between">
+        <div className="flex items-center gap-2">
+          <h1 className="heading-2 text-gray-900 capitalize truncate max-w-[15.625rem] sm:max-w-full">
+            {metadata?.displayName}
+          </h1>
+          <div className="flex items-center gap-2 text-gray-900 bg-base-bg rounded-lg p-[0.375rem]">
```
Comment (CzarekDryl) [inline src/components/frame/v5/pages/DashboardPage/partials/ColonyHeader/ColonyHeader.tsx:38]: `p-[0.375rem]` => `p-1.5`

File: src/routes/Routes.tsx [hunk 7]
```diff
@@ -121,7 +122,8 @@ const Routes = () => {
 
         {/* Colony routes */}
         <Route path={COLONY_HOME_ROUTE} element={<ColonyRoute />}>
-          <Route index element={<ColonyHome />} />
+          {/* <Route index element={<ColonyHome />} /> */}
```
Comment (KornelKwiatkowski) [inline src/routes/Routes.tsx:125]: Please, add a comment, why this line is commented

**Review by KornelKwiatkowski — APPROVED** [review]

File: src/components/frame/v5/pages/DashboardPage/partials/ColonyHeader/ColonyHeader.tsx [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+        {truncatedText ? (
+          <p className="text-gray-700 max-w-[52.75rem] text-md">
+            {truncatedText}
+          </p>
+        ) : (
+          <p className="text-gray-700">
+            {formatText({ id: 'colony.description' })}
+          </p>
+        )}
+        <Links />
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/ColonyHeader/ColonyHeader.tsx:59]: Please add a more descriptive name for this component and also add a wrapper here with `flex-shrink-0` classname

File: src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+    [ExternalLinks.Discord]: 'discord-logo',
+    [ExternalLinks.Github]: 'github-logo',
+    [ExternalLinks.Twitter]: 'twitter-logo',
+    [ExternalLinks.Telegram]: 'telegram-logo',
+    [ExternalLinks.Instagram]: 'instagram-logo',
+  };
+
+  return (
+    <div className="flex items-center gap-4">
+      {sortedLinks && (
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx:27]: Please check the length here

File: src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+import { useExternalLinks } from '~hooks/useExternalLinks';
+import ShareIcon from '../ShareIcon';
+
+const displayName = 'v5.pages.DashboardPage.partials.Links';
+
+const Links: FC = () => {
+  const { sortedLinks } = useExternalLinks();
+
+  const linkIconStyles = 'flex items-center md:hover:text-blue-400';
+  const iconMappings = {
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx:15]: Please move it to consts

File: src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+import { ExternalLinks } from '~gql';
+import { useExternalLinks } from '~hooks/useExternalLinks';
+import ShareIcon from '../ShareIcon';
+
+const displayName = 'v5.pages.DashboardPage.partials.Links';
+
+const Links: FC = () => {
+  const { sortedLinks } = useExternalLinks();
+
+  const linkIconStyles = 'flex items-center md:hover:text-blue-400';
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx:14]: Why is this added as a const here? I can see it used in one place only

File: src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+                    name={iconMappings[name] || ''}
+                    appearance={{ size: 'small' }}
+                  />
+                </Link>
+              </li>
+            ))
+            .slice(0, 3)}
+        </ul>
+      )}
+      <ShareIcon />
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Links/Links.tsx:43]: Same for those two components (naming)

File: src/components/frame/v5/pages/DashboardPage/partials/Menu/Menu.tsx [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+const Menu: FC = () => {
+  const { colony } = useColonyContext();
+  const { handleUnwatch } = useColonySubscription();
+
+  const {
+    getTooltipProps,
+    setTooltipRef,
+    setTriggerRef,
+    visible: isPopoverOpen,
+  } = usePopperTooltip({
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Menu/Menu.tsx:24]: Please also set close on outside click

File: src/components/frame/v5/pages/DashboardPage/partials/Menu/Menu.tsx [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+          tooltipProps={getTooltipProps}
+          withTooltipStyles={false}
+          cardProps={{
+            rounded: 's',
+            hasShadow: true,
+            className: 'py-4 px-2',
+          }}
+          classNames="w-full sm:max-w-[17.375rem]"
+        >
+          <Link
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Menu/Menu.tsx:51]: Can you create a component with groups of links or buttons and then use it here?

File: src/components/frame/v5/pages/DashboardPage/partials/MenuContentItem/MenuContentItem.tsx [hunk 15]
```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import Icon from '~shared/Icon';
+import { formatText } from '~utils/intl';
+import { MenuContentItemProps } from './types';
+
+const displayName = 'v5.pages.DashboardPage.partials.Menu';
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/MenuContentItem/MenuContentItem.tsx:6]: displayName is incorrect here,also this component should be redundant if you will add a component with groups of links/buttons

File: src/components/frame/v5/pages/DashboardPage/partials/ShareIcon/ShareIcon.tsx [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+import { useColonyContext } from '~hooks';
+import { useCopyToClipboard } from '~hooks/useCopyToClipboard';
+import Icon from '~shared/Icon';
+import Tooltip from '~shared/Extensions/Tooltip';
+import { formatText } from '~utils/intl';
+import { ShareIconProps } from './types';
+
+const displayName = 'v5.pages.DashboardPage.partials.ShareIcon';
+
+const ShareIcon: FC<PropsWithChildren<ShareIconProps>> = ({
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/ShareIcon/ShareIcon.tsx:12]: This component should be redundant too

File: src/components/v5/frame/PageLayout/PageLayout.tsx [hunk 17]
```diff
@@ -92,7 +92,7 @@ const PageLayout: FC<PropsWithChildren<PageLayoutProps>> = ({
                     {sidebar}
                   </div>
                 </div>
-                <div className="md:flex-grow flex flex-col gap-8">
+                <div className="md:flex-grow flex flex-col gap-[1.125rem]">
```
Comment (joanna-pagepro) [inline src/components/v5/frame/PageLayout/PageLayout.tsx:95]: Is this change needed? It will affect other pages than dashboard

File: src/components/v5/shared/BurgerMenu/types.ts [hunk 18]
```diff
@@ -1,4 +1,5 @@
 export type BurgerMenuProps = {
```
Comment (joanna-pagepro) [inline src/components/v5/shared/BurgerMenu/types.ts:1]: Please change it to interface

File: src/hooks/useExternalLinks.ts [hunk 19]
```diff
@@ -0,0 +1,28 @@
+import { ExternalLinks } from '~gql';
+import useColonyContext from './useColonyContext';
+
+export const useExternalLinks = () => {
```
Comment (joanna-pagepro) [inline src/hooks/useExternalLinks.ts:4]: Please add a return type here

File: src/images/icons.json [hunk 20]
```diff
@@ -228,7 +228,7 @@
     "bullets",
     "numbers",
     "quotes",
-    "rocket",
+    "rocket-logo",
```
Comment (joanna-pagepro) [inline src/images/icons.json:231]: What's with `rocket` icon? I can still see it in the project

File: src/routes/Routes.tsx [hunk 21]
```diff
@@ -121,7 +121,7 @@ const Routes = () => {
 
         {/* Colony routes */}
         <Route path={COLONY_HOME_ROUTE} element={<ColonyRoute />}>
-          <Route index element={<ColonyHome />} />
+          <Route index element={<DashboardPage />} />
```
Comment (joanna-pagepro) [inline src/routes/Routes.tsx:124]: Are you sure that this should replace `ColonyHome`?

**Review by joanna-pagepro — CHANGES_REQUESTED** [review]

File: src/hooks/useExternalLinks.ts (hunk 19 above)
Comment (joanna-pagepro) [reply src/hooks/useExternalLinks.ts:4]: It's also not something that will be used globally, so I think that you can move it to the directory where it's used

File: src/components/frame/v5/pages/DashboardPage/partials/ColonyHeader/ColonyHeader.tsx [hunk 22]
```diff
@@ -0,0 +1,67 @@
...
+import { formatText } from '~utils/intl';
+import { MAX_TEXT_LENGTH } from './consts';
+import { getCurrentToken } from '~common/ColonyTotalFunds/SelectedToken/helpers';
+import Icon from '~shared/Icon';
+import Tooltip from '~shared/Extensions/Tooltip';
+import Links from '../Links';
+
+const displayName = 'v5.pages.DashboardPage.partials.ColonyHeader';
+
+const ColonyHeader: FC = () => {
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/ColonyHeader/ColonyHeader.tsx:12]: ColonyHeader -> ColonyDashboardHeader

**Review by arrenv — CHANGES_REQUESTED** [review]
I'm not able to get the page to load at all due to the console errors as of the screenshot.

image

File: src/routes/Routes.tsx [hunk 23]
```diff
@@ -73,6 +72,7 @@ import ColonySettingsRoute from './ColonySettingsRoute';
 import UserRoute from './UserRoute';
 import WizardRoute from './WizardRoute';
 import LandingPageRoute from './LandingPageRoute';
+import ColonyHome from '~common/ColonyHome';
```
Comment (ArmandoGraterol) [inline src/routes/Routes.tsx:75]: Needs to go up to where it was

**Review by ArmandoGraterol — DISMISSED** [review]
- The chevron above is huge when in the design it's just 12x12px:
FireShot Capture 225 - Extensions - Colony - - localhost

- The hamburger menu icon is not blue when the menu is active:
FireShot Capture 226 - Extensions - Colony - - localhost

- The menu on mobile should have a space of 24px between the menu and the edge of the screen on both sides
FireShot Capture 227 - Extensions - Colony - - localhost

Specs:
FireShot Capture 228 - Colony V5 – Figma - www figma com

**Review by melyndav — CHANGES_REQUESTED** [review]
Hi @KatarzynaStefanowiczPagepro. Thanks for your work on this. The following updates are required.

1. The top padding between the title and header content does not match Figma for desktop.
image

2. At 1440px, the title should left aligned with the header content. 
image

3. The gap between the Colony title and token box doesn't match Figma.
image

4. The token widget should be bottom-aligned to the Colony title text as per Figma:
image

5. Padding between native token and lock icon doesn't match Figma. 
image

6. The kebab icon should stay text-blue-400 when the dropdown menu is activated:
image

7. The horizontal rules render different thickness?
image

8. The colony description at max characters should be 2 lines max as per Figma.
image

9. The share icon should copy the colonies URL. 

10. When social links are added the Youtube is showing without an icon.
image

11. A change has been made to the dropdown menu and the external links. When the dropdown menu is activated, all links including the 3 prioritises (website, github and whitepaper) should now show under the external links tab at all times. So everything is bunched together here as per here: 

image

Link to Figma: https://www.figma.com/file/5V8pr7iMwXsT9L3VAZsmUt/Colony-V5?type=design&node-id=11414-153565&mode=design&t=6jWe0sq1QWEq9jhU-4

12. When I hover over the Youtube icon, the URL is strange
image

13. When I select the external links in the kebab menu the menu breaks.
Animation12345

14. All icons are wrong size.
image

15. Leave a Colony doesn't work.

File: src/components/frame/v5/pages/DashboardPage/partials/GroupedMenuItems/GroupedMenuItems.tsx [hunk 24]
```diff
@@ -0,0 +1,105 @@
...
+
+  const iconStyles = '!w-[0.813rem] !h-[0.813rem]';
+
+  return (
+    <ul className="flex flex-col gap-[0.25rem]">
+      {list.map(({ items, key }) => {
+        return (
+          !!items?.length && (
+            <li key={key}>
+              {items.map(
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/GroupedMenuItems/GroupedMenuItems.tsx:25]: Please add another list inside (ul with li)

File: src/components/frame/v5/pages/DashboardPage/partials/Menu/types.ts [hunk 25]
```diff
@@ -0,0 +1,37 @@
...
+export const MENU_ITEM_TYPE = {
+  BUTTON: 'button',
+  BUTTON_WITH_TOOLTIP: 'button-with-tooltip',
+  LINK: 'link',
+  DIVIDER: 'divider',
+} as const;
+
+export type MenuItemType = (typeof MENU_ITEM_TYPE)[keyof typeof MENU_ITEM_TYPE];
+
+export interface MenuItemList {
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Menu/types.ts:12]: MenuItemList -> MenuItem
This is the pattern that you should use. The component's name should be related to Colony Dashboard, because it's in its partials

File: src/components/frame/v5/pages/DashboardPage/partials/Menu/types.ts [hunk 26]
```diff
@@ -0,0 +1,37 @@
...
+  itemKey: string;
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Menu/types.ts:13]: key

File: src/components/frame/v5/pages/DashboardPage/partials/Menu/types.ts [hunk 27]
```diff
@@ -0,0 +1,37 @@
...
+  additionalContent?: JSX.Element;
+  ariaLabel?: MessageDescriptor | string;
+  isSuccess?: boolean;
+  onClick?: () => void;
+  tooltipContent?: JSX.Element;
+  className?: string;
+}
+
+export interface MenuItemListProps {
+  key?: string;
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Menu/types.ts:29]: Key should be required

File: src/components/frame/v5/pages/DashboardPage/partials/Menu/types.ts [hunk 28]
```diff
@@ -0,0 +1,37 @@
...
+  items: MenuItemList[];
+}
+
+export interface UseMenuReturnType {
+  menuItems: MenuItemListProps[];
+  activeSubmenu: boolean;
```
Comment (joanna-pagepro) [inline src/components/frame/v5/pages/DashboardPage/partials/Menu/types.ts:35]: activeSubmenu sounds like you wanted to pass an array or a component and not a boolean

File: src/components/v5/frame/PageLayout/PageLayout.tsx [hunk 29]
```diff
@@ -20,10 +21,16 @@ const PageLayout: FC<PropsWithChildren<PageLayoutProps>> = ({
...
   children,
   hasWideSidebar,
+  colonyName,
 }) => {
   const wrapperRef = useRef<HTMLDivElement | null>(null);
   const topContentWrapperRef = useRef<HTMLDivElement | null>(null);
   const isTablet = useTablet();
+  const { pathname } = useLocation();
+
+  const isDashboardRoute = pathname
```
Comment (joanna-pagepro) [inline src/components/v5/frame/PageLayout/PageLayout.tsx:31]: Can you use `useMatch` hook instead?

File: src/components/v5/frame/PageLayout/types.ts [hunk 30]
```diff
@@ -7,4 +7,5 @@ export interface PageLayoutProps {
   headerProps: PageHeaderProps;
   topContent?: ReactNode;
   hasWideSidebar?: boolean;
+  colonyName?: string;
```
Comment (joanna-pagepro) [inline src/components/v5/frame/PageLayout/types.ts:10]: This shouldn't be needed here

File: src/components/v5/shared/BurgerMenu/BurgerMenu.tsx [hunk 31]
```diff
@@ -1,24 +1,32 @@
...
+        className,
+        'flex text-gray-400 sm:hover:text-blue-400 p-[0.1875rem] transition-all duration-normal cursor-pointer',
+      )}
       aria-label={formatMessage({ id: 'ariaLabel.openMenu' })}
       ref={setTriggerRef}
     >
       <Icon
         name={isVertical ? 'dots-three-vertical' : 'dots-three'}
-        appearance={{ size: 'extraTiny' }}
+        className="!w-[0.813rem] !h-[0.813rem]"
```
Comment (joanna-pagepro) [inline src/components/v5/shared/BurgerMenu/BurgerMenu.tsx:29]: This change is global/ Are you sure that those changes won't break anything?

File: src/hooks/useExternalLinks.ts [hunk 32]
```diff
@@ -0,0 +1,31 @@
+import { ExternalLinks } from '~gql';
+import useColonyContext from './useColonyContext';
+
+interface UseBalancePageReturnType {
+  name: ExternalLinks;
+  link: string;
+}
+
+export const useExternalLinks = (): UseBalancePageReturnType[] => {
```
Comment (joanna-pagepro) [inline src/hooks/useExternalLinks.ts:9]: This is not global. Please move it to the directory with dashboard components

File: src/hooks/useExternalLinks.ts [hunk 33]
```diff
@@ -0,0 +1,31 @@
...
+  const { colony } = useColonyContext();
+  const { metadata } = colony || {};
+  const { externalLinks } = metadata || {};
+
+  // @TODO: add facebook link
+  const linksPriority = [
```
Comment (joanna-pagepro) [inline src/hooks/useExternalLinks.ts:15]: I've already seen this array somewhere :)

File: src/images/icons.json (hunk 20 above)
Comment (joanna-pagepro) [reply src/images/icons.json:232]: Yes, but you should add another icon and not edit the existing one

**Review by joanna-pagepro — CHANGES_REQUESTED** [review]

File: src/images/icons.json [hunk 34]
```diff
@@ -260,7 +260,16 @@
...
     "chart-line",
     "presentation-chart",
-    "book-open-text"
+    "book-open-text",
+    "custom-logo",
+    "whitepaper-logo",
+    "bell-ringing",
+    "door",
+    "rocket",
+    "rocket-logo",
```
Comment (ArmandoGraterol) [inline src/images/icons.json:269]: This one is a duplicate, `rocket-logo` is already included above

**Review by ArmandoGraterol — COMMENTED** [review]
Thank you for the changes! I'm still noticing that the menu hamburger icon doesn't stay blue when the menu is active:

Screenshot 2023-11-23 at 12 40 52

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @krawczykkatarzyna thanks for your work on this! A few more updates needed:

1. Social links are missing some icons under the external links menu.
image

2. Discord should also show up the top as the three that have main priority - 

image

3. These three icons are missing text-blue-400 hover states
image

4. The padding needs to match with Figma between the header and the title
image
 
 5. The issue of the external links text in the sub-menu of the dropdown is still appearing. You can see the text fly off to the left side.
 
Animation11122

6. When I use the share feature, the green tooltip with the check appears (correct state) but then after a few seconds, it changes to the black default version. This shouldn't happen.

image

7. Leave Colony function still doesn't work. 

8. Can we center align the grey widget with the native token information with the Colony title? It's not appearing great so will be an easier fix.

image

9. Icons are still the wrong size. Please use the phosphor icons too. 

image

10. When you hover over an item in the dropdown menu, it takes a moment for the previous item hovered to de-bold. So there is a delay on the hover state change. 

11. Hover state is missing on the inside dropdown menu like the top hamburger style - 
image

File: src/components/common/ColonyHome/ColonyHome.tsx [hunk 35]
```diff
@@ -71,7 +69,8 @@ const ColonyHome = () => {
   };
 
   return (
-    <div className="flex flex-col gap-6">
+    <div className="flex flex-col gap-[2.5rem]">
```
Comment (ArmandoGraterol) [inline src/components/common/ColonyHome/ColonyHome.tsx:72]: We can use `gap-10` here which is the equivalent to `2.5rem` in tailwind

File: src/components/v5/common/ColonyDashboardHeader/partials/ColonyLinks/ColonyLinks.tsx [hunk 36]
```diff
@@ -0,0 +1,73 @@
+import React, { FC } from 'react';
+import { ShareNetwork } from 'phosphor-react';
+import { useLocation } from 'react-router-dom';
+import { useCopyToClipboard } from '~hooks/useCopyToClipboard';
+import { COLONY_LINK_CONFIG } from '~constants/colonyLinks';
+import { formatText } from '~utils/intl';
+import Tooltip from '~shared/Extensions/Tooltip';
+import Link from '~v5/shared/Link';
+import Menu from '../Menu';
+import { useExternalLinks } from '../Menu/hooks';
```
Comment (ArmandoGraterol) [inline src/components/v5/common/ColonyDashboardHeader/partials/ColonyLinks/ColonyLinks.tsx:10]: We could add an export to the `index` file of `Menu` and that way we avoid having to write 2 imports for code that is on the same folder: `import Menu, { useExternalLinks } from '../Menu';`. Cleaner this way 👌

File: src/components/v5/common/ColonyDashboardHeader/partials/GroupedMenuItems/GroupedMenuItems.tsx [hunk 37]
```diff
@@ -0,0 +1,134 @@
...
+import Link from '~v5/shared/Link';
+import { GroupedMenuItemsProps } from './types';
+
+const displayName = 'v5.common.ColonyDashboardHeader.partials.GroupedMenuItems';
+
+const GroupedMenuItems: FC<GroupedMenuItemsProps> = ({ list }) => {
+  const iconStyles = '!w-[0.913rem] !h-[0.913rem]';
+
+  return (
+    <ul className="flex flex-col gap-[0.25rem]">
```
Comment (ArmandoGraterol) [inline src/components/v5/common/ColonyDashboardHeader/partials/GroupedMenuItems/GroupedMenuItems.tsx:15]: Also `gap-1`

File: src/components/v5/common/ColonyDashboardHeader/partials/Menu/hooks.tsx [hunk 38]
```diff
@@ -0,0 +1,209 @@
...
+          url: item.link,
+          iconName: item.name,
+          title: item.name,
+        })),
+      },
+    ],
+    [sortedLinks],
+  );
+
+  return { menuItems, isSubmenuActive, submenuItems };
```
Comment (ArmandoGraterol) [inline src/components/v5/common/ColonyDashboardHeader/partials/Menu/hooks.tsx:208]: Typescript errors here according to github

**Review by rdig — DISMISSED** [review]
- Current iteration stretches the width of the page underneath the colony switcher
Screenshot from 2023-11-26 04-29-23

- I believe the external links are not sorted properly based on `iconMappings`
Screenshot from 2023-11-26 04-33-57

- The external links submenu in the kebab dropdown, does not reset when closing the dropdown
Screenshot from 2023-11-26 04-32-28

**Review by joanna-pagepro — APPROVED** [review]

**Comment by melyndav** [comment]
Hey @krawczykkatarzyna a new issue I mentioned in Slack, the colony name should show in the first breadcrumb and not the slug. As you can see the slug is showing here was 'Wayne' and not the full colony name which should show 'Wayne Enterprises'

image

**Comment by krawczykkatarzyna** [comment]
@rdig I can't reproduce that but with "Current iteration stretches the width of the page underneath the colony switcher". Can you check that again please?

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @krawczykkatarzyna -

A few issues:

1. The display is broken.
image

2. Leave Colony still doesn't work. The join button should show top right in the header when the user has left the Colony. 

Example of join: 
image


3. The share link that shows by default just copied [object Object] to my clipboard:
image

**Comment by krawczykkatarzyna** [comment]
@rdig @arrenv  @ArmandoGraterol  @melyndav  reporeted issue fixed

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @krawczykkatarzyna appreciate the changes.

When the user has left the colony they shouldn't see the option to leave the colony in the dropdown menu again. It should only show when they have joined the colony via the top right button. 

image

**Comment by krawczykkatarzyna** [comment]
@melyndav button has been hidden

**Review by melyndav — DISMISSED** [review]
Thanks for the fixes @krawczykkatarzyna approved. :)

**Review by arrenv — APPROVED** [review]
All looks good from here! 

There is a reloading when joining and leaving, but, I think it can be reviewed outside of this PR.

**Review by melyndav — APPROVED** [review]

**Review by CzarekDryl — APPROVED** [review]
